frappe/desk/query_report.py

```diff
@@ -185,7 +185,8 @@ def run(report_name, filters=None, user=None, ignore_prepared_report=False):
 	else:
 		result = generate_report_result(report, filters, user)
 
-	result["add_total_row"] = report.add_total_row and not result['skip_total_row']
+	skip_total_row = result['skip_total_row'] if 'skip_total_row' in result else ''
+	result["add_total_row"] = report.add_total_row and not skip_total_row
```

Comment:
```suggestion
	result["add_total_row"] = report.add_total_row and not result.get('skip_total_row')
```

Reply:
No need to store in a separate variable

Reply:
Thanks @deepeshgarg007  for Reviewing PR, can i proceed for Hotfix branch also.